fdbclient/Knobs.cpp

```diff
@@ -133,6 +133,10 @@ void ClientKnobs::initialize(bool randomize) {
 	init( BACKUP_RANGE_TIMEOUT,   TASKBUCKET_TIMEOUT_VERSIONS/CORE_VERSIONSPERSECOND/2.0 );
 	init( BACKUP_RANGE_MINWAIT,   std::max(1.0, BACKUP_RANGE_TIMEOUT/2.0));
 	init( BACKUP_SNAPSHOT_DISPATCH_INTERVAL_SEC,  10 * 60 );  // 10 minutes
+	init( BACKUP_INIT_SNAPSHOT_INTERVAL_SEC,          0);  // The initial snapshot has a desired duration of 0, meaning go as fast as possible.
+	if (randomize && BUGGIFY) {
```

Comment:
I think we are not using clang format for the knobs.cpp.  Instead, we keep one knob per line.
Please be consistent with the other knob's format in this file.

Reply:
Thanks. Done.